internal/pkg/cmd/prerunner.go

```diff
@@ -70,6 +71,7 @@ type AuthenticatedCLICommand struct {
 	V2Client          *ccloudv2.Client
 	MDSClient         *mds.APIClient
 	MDSv2Client       *mdsv2alpha1.APIClient
+	HubClient         *hub.Client
```

Comment:
We have a lot of custom clients here that shouldn't be initialized for every authenticated command: Kafka REST, Schema Registry, Metrics, Flink (soon), and now Confluent Hub. I think Kafka REST is the only one to do this, but they should only be initialized before we use each of them (as a singleton). We should also try to converge on some shared code pattern for all of these.